### PR TITLE
Restore a basic /community/who-uses-dart page

### PR DESCRIPTION
**Pull request by kwalrath**
We get lots of 404s for this page, but we don't have much to point people to. (Not that there aren't lots of people using Dart, but we don't have a system for showing off the non-Flutter uses.) I'm adding this page to avoid the 404s, but I'm not (yet) linking to it from anywhere.

/cc @messi01

### CODE REVIEW

**Comment by kwalrath** [comment]
Contributes to #679

**Comment by kwalrath** [comment]
Staged at https://kw-www-dartlang-1.firebaseapp.com/community/who-uses-dart. But do check out the source code, too (for social sharing text, e.g.).

File: src/community/who-uses-dart.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: Who uses Dart
+description: Mobile apps and Google web apps are two of the most common uses for the Dart language.
+toc: false
+---
+
+Many Dart developers write mobile apps for Android, iOS, or both,
```
Comment (mit-mit) [inline src/community/who-uses-dart.md]: I think we have to clarify the connection between Dart and Flutter in this statement

File: src/community/who-uses-dart.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+using the [Flutter toolkit.][Flutter]
+To find a list of Flutter apps, visit the [Flutter showcase.][showcase]
+
+[Flutter]: {{site.flutter}}
+[showcase]: {{site.flutter}}/showcase
+
+Google engineers use Dart to create many apps,
+including some that are essential to Google's business.
+For example, if you use the [Google Ads][] web or mobile app,
```
Comment (mit-mit) [inline src/community/who-uses-dart.md]: Maybe also mention Assistant on Smart Displays with a link to https://developers.googleblog.com/2019/05/Flutter-io19.html ?

File: src/community/who-uses-dart.md (hunk 1 above)
Comment (kwalrath) [reply src/community/who-uses-dart.md]: Maybe:

The popular Flutter toolkit relies on the Dart language, so developers writing Flutter apps — for Android, iOS, or other targets — use Dart.

File: src/community/who-uses-dart.md (hunk 2 above)
Comment (kwalrath) [reply src/community/who-uses-dart.md]: How about adding this sentence:

Also, the Assistant team at Google uses Dart for features in Smart Displays, as described in [this announcement.](https://developers.googleblog.com/2019/05/Flutter-io19.html)

File: src/community/who-uses-dart.md (hunk 1 above)
Comment (mit-mit) [reply src/community/who-uses-dart.md]: 👍

File: src/community/who-uses-dart.md (hunk 2 above)
Comment (mit-mit) [reply src/community/who-uses-dart.md]: 👍

**Comment by kwalrath** [comment]
ptal. I've updated the staging server: https://kw-www-dartlang-1.firebaseapp.com/community/who-uses-dart

**Pull request merged**